subsys/partition_manager/CMakeLists.txt

```diff
@@ -78,7 +78,16 @@ if (CONFIG_BUILD_WITH_TFM)
     set_property(GLOBAL PROPERTY
       bl2_PM_HEX_FILE ${CMAKE_BINARY_DIR}/tfm/bin/bl2.hex
       )
+    set_property(GLOBAL PROPERTY
```

Comment:
This (everything itside the `if (CONFIG_BUILD_WITH_TFM)` should really all be moved out to some TFM specific subsys folder CMakeLists.txt. Its not something you work on directly in this PR, but it would be great if you could handle it here.

Reply:
Moved to subsys/tfm/CMakeLists.txt